CONTRIBUTING.md

```diff
@@ -31,6 +31,14 @@ To make this file used by `git blame`, you need to run the following command.
 git config --local blame.ignoreRevsFile .git-blame-ignore-revs
 ```
 
+## docs
+
+For local development check out the documentation:
+
+1. Run `pnpm run build` in Vite's root folder.(Since `twoslash` is configured in the document, to prevent errors, perform a global build before starting the document)
+
+2. Run `pnpm run docs` in Vite's root folder..
```

Comment:
```suggestion
## Documentation

To develop the `docs/` site:

1. Run `pnpm run build` in Vite's root folder. This will generate the types for `twoslash` to work in the code examples. If the types are not available, errors will be logged in step 2 but does not prevent the site from working.

2. Run `pnpm run docs` in Vite's root folder.
```

tweaked a bit

Reply:
Thank you! 👍